src/io/flutter/sdk/FlutterSettingsConfigurable.java

```diff
@@ -142,24 +113,22 @@ public void apply() throws ConfigurationException {
 
   @Override
   public void reset() {
-    isModified = false;
-
     final FlutterSdk sdk = FlutterSdk.getGlobalFlutterSdk();
-    final String path = sdk != null ? FileUtilRt.toSystemDependentName(sdk.getHomePath()) : "";
+    final String path = sdk != null ? sdk.getHomePath() : "";
     sdkCombo.getComboBox().getEditor().setItem(path);
 
     updateVersionText();
     updateErrorLabel();
   }
 
   private void updateVersionText() {
-
     final FlutterSdk sdk = FlutterSdk.forPath(getSdkPathText());
     if (sdk == null) {
       versionDetails.setVisible(false);
     }
     else {
       try {
+        final ModalityState modalityState = ModalityState.current();
```

Comment:
Sorry.  Not seeing that tab but I'm probably not looking in the right place...

<img width="1666" alt="screen shot 2016-10-20 at 6 02 38 am" src="https://cloud.githubusercontent.com/assets/67586/19560896/25efcf0c-968b-11e6-8243-197ef83162d0.png">

Time for more coffee! ☕️